packages/bpk-component-scrollable-calendar/stories.js

```diff
@@ -20,7 +20,7 @@ import { storiesOf } from '@storybook/react';
 
 import {
   DefaultExample,
-  WeekStartsOnSeven,
+  WeekStartsOnSix,
```

Comment:
Is this correct? Used to be `Seven` and now it's `Six`?

Reply:
I left a comment elsewhere about this – date-fns now throws an error if you pass in anything other than `0-6` for this, so I changed it.